File: packages/emitter/testcases/classes/getters-setters/GettersSetters.ts
export class Rectangle {
  private _width: number = 0;
  private _height: number = 0;

  get width(): number {
    return this._width;
  }

  set width(value: number) {
    this._width = value;
  }

  get area(): number {
    return this._width * this._height;
  }
}

File: packages/emitter/testcases/classes/getters-setters/GettersSetters.cs
using Tsonic.Runtime;

namespace TestCases.classes
{
    public class Rectangle
    {
        private double _width = 0.0;
        private double _height = 0.0;

        public double width
        {
            get
            {
            return this._width;
            }
            set
            {
            this._width = value;
            }
        }

        public double area
        {
            get
            {
            return this._width * this._height;
            }
        }
    }

    public static class GettersSetters
    {
    }
}
